docs/development/releasing/index.md

```diff
@@ -1,42 +1,53 @@
 # Releasing
 
-## Helm Charts
+As prerequisites to release:
+
+- The repository that holds the code for the component/package to release has:
+  - A release commit as last commit in `master` branch
+  - The last commit in master is tagged.
 
-### 1. Build and test locally:
+## Python packages
+
+**1.** Release the package(s)
 
 ```console
-$ reana-dev run-ci \
-    --admin-email john.doe@example.org \
-    --admin-password 123456 \
-    --component r-d-r-roofit \
-    --exclude-components=r-ui,r-m-broker
+$ reana-dev release-pypi -c <package-name>
 ```
 
-### 2. Build and test using the images to be released:
+## Docker images
+
+**1.** Clean your environment
 
 ```console
-$ reana-dev run-ci \
-    --admin-email john.doe@example.org \
-    --admin-password 123456 \
-    --mode releasehelm  # FIXME
+$ reana-dev git-clean
 ```
 
-### 3. Upgrade chart version and commit:
+**2.** Build the image(s)
 
-If everything goes well you will see that the REANA components images in `helm/reana/values.yaml` are updated. Now you can upgrade the version in the `helm/reana/Chart.yaml` file, push your changes and create a pull request.
+```console
+$ reana-dev docker-build -c <component-name> --no-cache
+```
 
-```diff
-$ git diff helm/reana/Chart.yaml
--version: 0.7.0-dev20200416
-+version: 0.7.0
-$ git commit
+**3.** Release the image(s)
+
+```console
+$ reana-dev release-docker -c <component-name>
 ```
 
-!!! info
-    The release will be created and added to the [REANA releases](https://github.com/reanahub/reana/releases) once the pull request is merged using the GitHub action [Helm Chart Releaser](https://github.com/marketplace/actions/helm-chart-releaser) in [`helm-releaser.yaml`](https://github.com/reanahub/reana/blob/master/.github/workflows/helm-releaser.yaml).
+Note: repeat `2.` and `3.` until you release all the images you need.
```

Comment:
I would remove this note as it is kind of obvious... Otherwise people can use be better learn about `-c r-component1 -c r-component1`...